### PR TITLE
DROOLS-1781: replaces findbugs-maven-plugin

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
Jenkins please retest this

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/pom.xml [hunk 1]
```diff
@@ -19,6 +19,21 @@
...
   </properties>
 
+  <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
+          <configuration>
+            <excludeFilterFile>${project.basedir}/../../../src/main/spotbugs/spotbugs-exclude.xml</excludeFilterFile>
```
Comment (mareknovotny) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/pom.xml]: look at my conversation with @gitgabrio at https://github.com/kiegroup/drools-wb/pull/1015#discussion_r259234514, the filter can be removed completely.

**Comment by mbiarnes** [comment]
@mareknovotny done so far - but one filter is still there

**Comment by mareknovotny** [comment]
@mbiarnes why is there still one filter? I thought we can remove that too as TitledAttachmentFileWidget constructor which throws that would be deleted?

**Comment by mbiarnes** [comment]
Jenkins please retest this

**Pull request merged**